styles/all/template/wpn_acp.js

```diff
@@ -0,0 +1,79 @@
+/* global phpbb */
```

Comment:
This JS file, if only used in the ACP, could be put in the arm/style folder instead